shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/screens/AboutConference.kt

```diff
@@ -1,21 +1,211 @@
+import org.jetbrains.kotlinconf.Speaker
+import org.jetbrains.kotlinconf.SpeakerId
+import org.jetbrains.kotlinconf.ui.components.DayHeader
+import org.jetbrains.kotlinconf.ui.components.Divider
+import org.jetbrains.kotlinconf.ui.components.PageMenuItem
 import org.jetbrains.kotlinconf.ui.components.StyledText
+import org.jetbrains.kotlinconf.ui.theme.KotlinConfTheme
 
 @Composable
-fun AboutConference() {
-    val navController = LocalNavController.current
-    Column {
-        Image(painterResource(Res.drawable.arrow_left_24), "back", modifier = Modifier.clickable { navController.popBackStack() })
-        StyledText("About Conference")
+fun AboutConference(
+    onPrivacyPolicy: () -> Unit,
+    onGeneralTerms: () -> Unit,
+    onWebsiteLink: () -> Unit,
+    onBack: () -> Unit,
+) {
+    ScreenWithTitle(
+        title = stringResource(Res.string.about_conference_title),
+        onBack = onBack,
+    ) {
+        Column(
+            modifier = Modifier.padding(top = 24.dp),
+            verticalArrangement = Arrangement.spacedBy(24.dp),
+        ) {
+            StyledText(text = stringResource(Res.string.about_conference_header), style = KotlinConfTheme.typography.h1)
+            StyledText(text = stringResource(Res.string.about_conference_timeline))
+            StyledText(text = stringResource(Res.string.about_conference_description))
+            StyledText(text = stringResource(Res.string.about_conference_hashtag))
+        }
+
+        Spacer(modifier = Modifier.height(48.dp))
+
+        Column(
+            verticalArrangement = Arrangement.spacedBy(48.dp),
+        ) {
+            // TODO retrieve data from the service
+            Event(
+                month = "MAY",
+                day = "22",
+                line1 = stringResource(Res.string.about_conference_day1_title_line1),
+                line2 = stringResource(Res.string.about_conference_day1_title_line2),
+                description = "",
+                speakers = listOf(
+                    Speaker(SpeakerId("0"), "Hadi Hariri", "", "Never believed in elevator pitches", ""),
+                    Speaker(SpeakerId("1"), "Hadi Hariri", "", "Never believed in elevator pitches", ""),
+                ),
+                location = "Hall A",
+                time = "10:00 – 11:00",
+            )
+
+            Event(
+                month = "MAY",
+                day = "23",
+                line1 = stringResource(Res.string.about_conference_day2_title_line1),
+                line2 = stringResource(Res.string.about_conference_day2_title_line2),
+                description = "",
+                speakers = listOf(
+                    Speaker(SpeakerId("2"), "Hadi Hariri", "", "Never believed in elevator pitches", ""),
+                ),
+                location = "Hall A",
+                time = "10:00 – 11:00",
+            )
+
+            Event(
+                month = "MAY",
+                day = "22",
+                day2 = "–23",
```

Comment:
This is a really annoying thing to have to add, I didn't realize this component might need to display two days 😅 

Suggestion: let's remove the dash from here, and add that within the component itself.

Also, this is a small thing, but the design has a `-` in it while this here is a `–`.